exercises/calculator.cpp

```diff
@@ -10,3 +10,39 @@
   Multiplication: 8
   Division: 2
 */
+
+double sum(double a, double b){
+  return a+b;
+}
+
+double difference(double a, double b){
```

Comment:
potresti passare i parametri per riferimento in modo da risparmiare spazio in memoria
```diff
+ double sum(double& a, double& b){
- double sum(double a, double b){
```
Vale anche per gli altri metodi!